addons/hr/models/hr_employee_base.py

```diff
@@ -92,7 +92,7 @@ def _compute_last_activity(self):
             tz = employee.tz
             last_presence = presences.get(employee.user_id.id, False)
             if last_presence:
-                last_activity_datetime = last_presence.replace(tzinfo=UTC).astimezone(timezone(tz)).replace(tzinfo=None)
+                last_activity_datetime = last_presence.replace(tzinfo=UTC).astimezone(timezone(tz))
                 employee.last_activity = last_activity_datetime.date()
                 if employee.last_activity == fields.Date.today():
                     employee.last_activity_time = format_time(self.env, last_activity_datetime, time_format='short')
```

Comment:
```suggestion
                    employee.last_activity_time = format_time(self.env, last_presence, tz=tz, time_format='short')
```

Reply:
what you are suggesting is already done in the format_time function

Reply:
The issue is that we give a localised time to `format_time` prentending it's not (removing the `tzinfo`), so it's converted twice. To solve your issue, you just need to give the time in UTC, not to change the tools method.

Reply:
ah alright I will fix that, Thank you